irteus/bullet.l

```diff
@@ -0,0 +1,122 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;
+;;; $Id$
+;;;
+;;; Copyright (c) 1987- JSK, The University of Tokyo.  All Rights Reserved.
+;;;
+;;; This software is a collection of EusLisp code for robot applications,
+;;; which has been developed by the JSK Laboratory for the IRT project.
+;;; For more information on EusLisp and its application to the robotics,
+;;; please refer to the following papers.
+;;;
+;;; Toshihiro Matsui
+;;; Multithread object-oriented language euslisp for parallel and
+;;;  asynchronous programming in robotics
+;;; Workshop on Concurrent Object-based Systems,
+;;;  IEEE 6th Symposium on Parallel and Distributed Processing, 1994
+;;;
+;;; Permission to use this software for educational, research
+;;; and non-profit purposes, without fee, and without a written
+;;; agreement is hereby granted to all researchers working on
+;;; the IRT project at the University of Tokyo, provided that the
+;;; above copyright notice remains intact.
+;;;
+
+(require :irtmath)
+(require :irtgeo)
+
+(in-package "GEOMETRY")
+
+(export '(bt-collision-distance bt-collision-check))
+
+(defun bt-make-model-from-body
+    (b &key (csg (send b :csg)) (margin nil) m)
+  "Make bullet model from body."
+  (cond ((assoc :sphere csg)
+         (setq m
+               (btmakespheremodel
+                (* 1e-3 (user::radius-of-sphere b)))
+               ))
+        ((assoc :cube csg)
+         (setq m
+               (btmakeboxmodel
+                (* 1e-3 (user::x-of-cube b))
+                (* 1e-3 (user::y-of-cube b))
+                (* 1e-3 (user::z-of-cube b))
+                )))
+        ((assoc :cylinder csg)
+         (setq m
+               (btmakecylindermodel
+                (* 1e-3 (user::radius-of-cylinder b))
+                (* 1e-3 (user::height-of-cylinder b))
+                )))
+        (t
+         (setq m
+               (btmakemeshmodel
+                (scale 1e-3 ;; [m]
+                       (apply #'concatenate float-vector
+                              (mapcar #'(lambda (v) (send b :inverse-transform-vector v)) (send b :vertices))))
+                (length (send b :vertices))
+                ))
+         ))
+  (when margin
+    (btsetmargin m margin))
+  m)
+
+(defmethod cascaded-coords
+  (:make-btmodel
+   (&key (fat 0) vs m)
```

Comment:
Arguments ```ms``` and ```m``` are local variables.
Please consider https://github.com/mmurooka/jskeus/pull/5

Reply:
Thanks, merged.